.github/workflows/proto-buf-publisher.yml

```diff
@@ -0,0 +1,38 @@
+name: Proto Buf Publishing - Action
```

Comment:
i don't think that we need to publish our protobuf, but we need a linter for our proto files. thanks!

Reply:
You're probably going to want to publish them for easier interoperability